obs2ioda-v2/README.md

```
@@ -26,12 +26,21 @@ If you have an environment preconfigured for `mpas-jedi`, simply source that env
    ```bash
    cmake <OBS2IODA_ROOT_DIR> -DNCEP_BUFR_LIB=<NCEP_BUFR_LIB_PATH> -DCMAKE_BUILD_TYPE=<BUILD_TYPE>
    ```
+   For a detailed comparison of the supported build types, refer to the [Build Types Table.](#cmake-build-types-table)
 5. Finally, build `obs2ioda` using `CMake`'s build tool. In this case, we use `GNU Make`, but other build tools supported by `CMake` can be used:
    ```bash
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)